landlab/components/stream_power/stream_power.py

```diff
@@ -568,7 +568,7 @@ def erode(self, grid, dt, elevs='topographic__elevation',
             threshdt = self.sp_crit * dt
 
         # solve using Brent's Method in Cython for Speed
-        if type(threshdt) == float:
+        if isinstance(threshdt, float):
```

Comment:
Looks good. We should never be using `type(...) == ...`. I've tried to remove them when I see them but I'm sure there's more still.